src/components/LHNOptionsList/OptionRowLHNData.tsx

```diff
@@ -40,11 +41,11 @@ function OptionRowLHNData({
         // Note: ideally we'd have this as a dependent selector in onyx!
         const item = SidebarUtils.getOptionData({
             report: fullReport,
-            reportActions,
             personalDetails,
             preferredLocale: preferredLocale ?? CONST.LOCALES.DEFAULT,
             policy,
             parentReportAction,
+            reportErrors,
```

Comment:
This is not added to the dependencies of `useMemo`

Reply:
`reportErrors` seems to be an object, if we add it as dependency we should memoize it also on the parent component

parent:
```
const reportErrors = useMemo(() => reportIDsWithErrors[reportID] ?? {}, [reportID]);
```
and not sure about `reportIDsWithErrors` 
cc: @youssef-lr